planner/core/logical_plans_test.go

```diff
@@ -67,8 +57,9 @@ func (s *testUnitTestSuit) SubstituteCol2CorCol(expr expression.Expression, colI
 	return expr, nil
 }
 
-func (s *testUnitTestSuit) TestIndexPathSplitCorColCond(c *C) {
-	defer testleak.AfterTest(c)()
+func TestIndexPathSplitCorColCond(t *testing.T) {
+	defer testleak.AfterTestT(t)()
```

Comment:
We're using goleak for leak tests, see also other tests for example https://github.com/pingcap/tidb/pull/27277.

Reply:
thanks for letting me know! removed this line because goleak is invoked in TestMain for this package already.